src/s3ql/__init__.py

```diff
@@ -38,8 +38,11 @@
 
 from llfuse import ROOT_INODE
 
+# The RELEASE information is built from 'VERSION RELEASE_SUFFIX'
+# E.g., if VERSION='3.1' and RELEASE_SUFFIX='(beta)', RELEASE will be '3.0 (beta)'
 VERSION = '3.0'
-RELEASE = '%s' % VERSION
+RELEASE_SUFFIX = ''
```

Comment:
Hmm. Why the new `RELEASE_SUFFIX`? I think if this is going to be needed at some point in the future, we can still introduce it then.

Reply:
I see. I thought it would be needed since it seems that at least once, `VERSION` and `RELEASE` differed (see [here](https://github.com/s3ql/s3ql/blame/6c6825527d38050f4141c0b266aacbd5dce92f3b/src/s3ql/__init__.py#L19)). But I agree it would be easier to only go with `VERSION` and avoid assembling `RELEASE` from `VERSION` and a suffix which would be empty most of the time.

Is this what you prefer?